independent-publisher-connectors/Mensagia/apiProperties.json

```diff
@@ -1,25 +1,25 @@
 {
-  "properties": {
-    "connectionParameters": {
-      "api_key": {
-        "type": "securestring",
-        "uiDefinition": {
-          "displayName": "Api Key",
-          "description": "The API Key for this api",
-          "tooltip": "Provide your API Key",
-          "constraints": {
-            "tabIndex": 2,
-            "clearText": false,
-            "required": "true"
-          }
-        }
-      }
-    },
-    "iconBrandColor": "#da3b01",
-    "capabilities": [
-      "actions"
-    ],
-    "publisher": "Sistemas Informaticos ICON, S.L.",
-    "stackOwner": "Sinergyne Global Communications, S.L."
-  }
-}
+    "properties": {
+        "connectionParameters": {
+            "api_key": {
+                "type": "securestring",
+                "uiDefinition": {
+                    "displayName": "Api Key",
+                    "description": "Api Key for this API.",
+                    "tooltip": "Set your Api Key.",
+                    "constraints": {
+                        "tabIndex": 2,
+                        "clearText": false,
+                        "required": "true"
+                    }
+                }
+            }
+        },
+        "iconBrandColor": "#FDFDFD",
+        "capabilities": [
+            "actions"
+        ],
+        "publisher": "Mensagia",
```

Comment:
@Sistemes-Icon Can you please let us know the reason for changing it to "Mensagia" from your company name which is "Sistemas Informaticos ICON, S.L.". 

Please share with us any legal document or an email from Mensagia representative that you have the authority to publish this connector on behalf of "Mensagia" and that should be published as a "certified-connector", not an independent publisher connector.

Reply:
@sriyen-msft In this image you can see how Stefano (Mensagia's CEO) asks to change the company name.
We (Sistemas Informaticos ICON, S.L.) are making this connector for them. And they want their contact information to be there.
![image](https://user-images.githubusercontent.com/83464318/169470540-11f44cad-aeb1-477b-a338-35216dc792c0.png)